protocol/communities/community.go

```diff
@@ -685,6 +685,10 @@ func (o *Community) getMember(pk *ecdsa.PublicKey) *protobuf.CommunityMember {
 	return member
 }
 
+func (o *Community) GetMember(pk *ecdsa.PublicKey) *protobuf.CommunityMember {
+	return o.getMember(pk)
+}
```

Comment:
not sure that it is needed, I guess you can call o.getMember(pk) where you need this fn

Reply:
I can't use `getMember` in messenger_communities because lower case methods are private.